pingdom-checks/loan_guaranty.ping

```diff
@@ -24,3 +24,15 @@ pingdom save-check \
   -a group="loan-guaranty" \
   -a userids_csv="$STATUSPAGE_SANDBOX_LOAN_GUARANTY_V1" \
   -a integrationids_csv="$LOAN_GUARANTY_SLACK_ID,$PAGER_DUTY_DEPENDENCIES"
+
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-guaranty-remittance-health-check \
```

Comment:
The reason I went with plain "Guaranty Remittance" is because even though it belongs to LGY, that's how they'll be naming the API. See the following beta release in the dev portal: https://dev-developer.va.gov/explore/loanGuaranty

Reply:
What about `lgy-guarantee-remittance` if we just want to identify it's for LGY in the name?